SwiftGit2/Experimental/Buffer.swift

```diff
@@ -59,10 +59,41 @@ public extension Buffer {
 		return String(data: data, encoding: .utf8)
 	}
 	
+	func asStringRez() -> Result<String, NSError> {
```

Comment:
rename to asString(), remove old asString.
if you need optional use try? asString().get()